### PR TITLE
Add description for systemsettings new features

### PR DESCRIPTION
**Pull request by jinwangan**
[SystemSettings] add descriptions for new features.

  - SYSTEM_SETTINGS_KEY_DEVELOPER_OPTION_STATE
  - SYSTEM_SETTINGS_KEY_AUTOMATIC_TIME_UPDATE
  - feature descriptions about systemsettings

### CODE REVIEW

**Comment by journey2w** [comment]
@annie-samsung 
Please review just English sentence.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 1]
```diff
@@ -438,6 +443,11 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/sensor.tiltmeter.wakeup` | Specify this key, if the application requires the tilt sensor wake-up feature. | 2.2.1 |
 | `http://tizen.org/feature/sensor.ultraviolet` | Specify this key, if the application requires a ultraviolet sensor. | 2.3   |
 | `http://tizen.org/feature/sensor.wrist_up` | Specify this key, if the application requires a wrist up sensor. | 2.3   |
+| `http://tizen.org/feature/systemsetting.home_screen` | Specify this key, if the application requires a home screen. | 4.0   |
+| `http://tizen.org/feature/systemsetting.lock_screen` | Specify this key, if the application requires a home screen. | 4.0   |
+| `http://tizen.org/feature/systemsetting.incoming_call` | Specify this key, if the application requires incomming call and sound notification. | 4.0   |
+| `http://tizen.org/feature/systemsetting.notification_email` | Specify this key, if the application requires a email notification. | 4.0   |
```
Comment (annie-samsung) [inline docs/application/native/tutorials/details/app-filtering.md]: requires an email

File: docs/application/native/tutorials/details/app-filtering.md [hunk 2]
```diff
@@ -438,6 +443,11 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/sensor.tiltmeter.wakeup` | Specify this key, if the application requires the tilt sensor wake-up feature. | 2.2.1 |
 | `http://tizen.org/feature/sensor.ultraviolet` | Specify this key, if the application requires a ultraviolet sensor. | 2.3   |
 | `http://tizen.org/feature/sensor.wrist_up` | Specify this key, if the application requires a wrist up sensor. | 2.3   |
+| `http://tizen.org/feature/systemsetting.home_screen` | Specify this key, if the application requires a home screen. | 4.0   |
+| `http://tizen.org/feature/systemsetting.lock_screen` | Specify this key, if the application requires a home screen. | 4.0   |
```
Comment (annie-samsung) [inline docs/application/native/tutorials/details/app-filtering.md]: Shouldn't it be lock screen?
Specify this key, if the application requires a lock screen.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 3]
```diff
@@ -438,6 +443,11 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/sensor.tiltmeter.wakeup` | Specify this key, if the application requires the tilt sensor wake-up feature. | 2.2.1 |
 | `http://tizen.org/feature/sensor.ultraviolet` | Specify this key, if the application requires a ultraviolet sensor. | 2.3   |
```
Comment (annie-samsung) [inline docs/application/native/tutorials/details/app-filtering.md]: requires an ultraviolet

File: docs/application/native/tutorials/details/app-filtering.md [hunk 4]
```diff
@@ -290,6 +290,11 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
...
 | `http://tizen.org/feature/sensor.wrist_up` | Specify this key, if the application requires a wrist up sensor. | 2.3   |
+| `http://tizen.org/feature/systemsetting.home_screen` | Specify this key, if the application requires a home screen. | 4.0   |
+| `http://tizen.org/feature/systemsetting.lock_screen` | Specify this key, if the application requires a home screen. | 4.0   |
```
Comment (annie-samsung) [inline docs/application/native/tutorials/details/app-filtering.md]: Specify this key, if the application requires a lock screen.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 5]
```diff
@@ -290,6 +290,11 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/sensor.tiltmeter.wakeup` | Specify this key, if the application requires the tilt sensor wake-up feature. | 2.2.1 |
 | `http://tizen.org/feature/sensor.ultraviolet` | Specify this key, if the application requires a ultraviolet sensor. | 2.3   |
```
Comment (annie-samsung) [inline docs/application/native/tutorials/details/app-filtering.md]: an ultraviolet

File: docs/application/native/guides/device/settings.md [hunk 6]
```diff
@@ -177,8 +177,10 @@ The following table lists the available system setting keys.
 | `SYSTEM_SETTINGS_KEY_3G_DATA_NETWORK_ENABLED` | `bool`   | `GETTER, NOTIFIER`      | Indicates whether the 3G data network is enabled. |
 | `SYSTEM_SETTINGS_KEY_ACCESSIBILITY_TTS`  | `bool`   | `GETTER, NOTIFIER`      | Indicates whether the accessibility TTS is enabled on the device. |
 | `SYSTEM_SETTINGS_KEY_ADS_ID`             | `string` | `GETTER, NOTIFIER`      | Ads ID for each device.                  |
```
Comment (annie-samsung) [inline docs/application/native/guides/device/settings.md]: Is it the word ADD or Advertisements contraction (AD)?
Then change spelling to "Adds ID for each device."

File: docs/application/native/guides/device/settings.md [hunk 7]
```diff
@@ -177,8 +177,10 @@ The following table lists the available system setting keys.
...
+| `SYSTEM_SETTINGS_KEY_AUTOMATIC_TIME_UPDATE`  | `bool` | `GETTER, NOTIFIER`      | Updating time automatically.           |
```
Comment (annie-samsung) [inline docs/application/native/guides/device/settings.md]: Updates time automatically.

Comment (annie-samsung) [inline docs/application/native/guides/device/settings.md]: Updates time automatically.     


For clarity do you want consider saying; device time or system time something to make it more clear what time is updated?

File: docs/application/native/tutorials/details/app-filtering.md [hunk 8]
```diff
@@ -290,6 +290,11 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
...
 | `http://tizen.org/feature/sensor.wrist_up` | Specify this key, if the application requires a wrist up sensor. | 2.3   |
+| `http://tizen.org/feature/systemsetting.home_screen` | Specify this key, if the application requires a home screen. | 4.0   |
+| `http://tizen.org/feature/systemsetting.lock_screen` | Specify this key, if the application requires a home screen. | 4.0   |
+| `http://tizen.org/feature/systemsetting.incoming_call` | Specify this key, if the application requires incomming call and sound notification. | 4.0   |
+| `http://tizen.org/feature/systemsetting.notification_email` | Specify this key, if the application requires a email notification. | 4.0   |
```
Comment (annie-samsung) [inline docs/application/native/tutorials/details/app-filtering.md]: requires an email

File: docs/application/native/tutorials/details/app-filtering.md (hunk 1 above)
Comment (jinwangan) [reply docs/application/native/tutorials/details/app-filtering.md]: I fixed it.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 2 above)
Comment (jinwangan) [reply docs/application/native/tutorials/details/app-filtering.md]: I fixed it

File: docs/application/native/tutorials/details/app-filtering.md (hunk 3 above)
Comment (jinwangan) [reply docs/application/native/tutorials/details/app-filtering.md]: I fixed it.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 4 above)
Comment (jinwangan) [reply docs/application/native/tutorials/details/app-filtering.md]: I fixed it.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 5 above)
Comment (jinwangan) [reply docs/application/native/tutorials/details/app-filtering.md]: I fixed it.

File: docs/application/native/guides/device/settings.md (hunk 6 above)
Comment (jinwangan) [reply docs/application/native/guides/device/settings.md]: it's Advertisements contraction.
I fixed to "ADS ID".

File: docs/application/native/guides/device/settings.md (hunk 7 above)
Comment (jinwangan) [reply docs/application/native/guides/device/settings.md]: I fixed it

File: docs/application/native/tutorials/details/app-filtering.md (hunk 8 above)
Comment (jinwangan) [reply docs/application/native/tutorials/details/app-filtering.md]: I fixed it.

File: docs/application/native/guides/device/settings.md (hunk 6 above)
Comment (annie-samsung) [reply docs/application/native/guides/device/settings.md]: Thanks for clarifying, for clarity recommend changing it to "Advertisement ID for each device."

File: docs/application/native/tutorials/details/app-filtering.md (hunk 8 above)
Comment (annie-samsung) [reply docs/application/native/tutorials/details/app-filtering.md]: I am not able to see a new commit of the file, will you pushing the fixed the files as a new commit to this PR?

**Pull request merged**